include/vcpkg/sourceparagraph.h

```diff
@@ -70,9 +70,7 @@ namespace vcpkg
     struct DependencyOverride
     {
         std::string name;
-        std::string version;
-        int port_version = 0;
-        VersionScheme version_scheme = VersionScheme::String;
+        SchemedVersion version;
```

Comment:
@vicroms requests

```
Version version;
VersionScheme scheme;
```

for consistency with https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1892704